woocommerce/woocommerce-ios
# Products: Grouping design for product creation type sheet

Closes: #11361, #11211


## Description


This PR is the second part of the product type creation bottom sheet.

Specifically, it now groups the "Manual product creation" into three categories:

1. Standard
   - Physical product
   - Virtual product
   - Variable product
2. Subscription
   - Simple subscription
   - Variable subscription
3. Other
   - Grouped product
   - External product


## Testing instructions


1. Go to Products tab,
2. Create new product,
3. Ensure that the bottom sheet demo matches the video below, and with the design video on https://github.com/woocommerce/woocommerce-ios/issues/11361#issue-2027620643
4. Additionally, test with a site that doesn't have subscriptions support. Ensure that the subscriptions category and its product types are not shown.

## Video



https://github.com/woocommerce/woocommerce-ios/assets/266376/4f2340dd-781b-4a76-9de3-1d1d93b09fe1


---
